lib/archive-utils.rb

```diff
@@ -20,5 +15,4 @@ module Archive
 require 'archive/fixity'
 require 'archive/operating_system'
 require 'archive/tarfile'
-include Archive
```

Comment:
Not sure what the implications of this could be for projects using this gem.  That will need to be evaluated before merging this PR.  That is, any projects using this will need to either specify a github branch dependency or checkout this branch and use a local path dependency, then run all their specs against it.